cmake/Anope.cmake

```diff
@@ -449,7 +449,8 @@ macro(calculate_libraries SRC SRC_LDFLAGS EXTRA_DEPENDS)
       if(DEFAULT_LIBRARY_DIRS OR WSDK_PATH OR DEFINED $ENV{VCINSTALLDIR})
         find_library(FOUND_${LIBRARY}_LIBRARY NAMES ${LIBRARY} PATHS ${DEFAULT_LIBRARY_DIRS} ${WSDK_PATH}/lib $ENV{VCINSTALLDIR}/lib ${EXTRA_INCLUDE} ${EXTRA_LIBS})
```

Comment:
I'm don't have a setup to test on Windows (which I'm assuming that code path is for) which is why I restricted my changes to only the path that my system was following. The additional changes would make sense, but absent testing I figured conservative was better.